### PR TITLE
update Metadata to include narwhal fields

### PR DESCRIPTION
**Pull request by laura-makdah**
Adds the fields required to create the narwhal committee at a new epoch to the validator registration function, and uses them to populate them on the Validator Metadata in the system state object.

### CODE REVIEW

**Review by lxfind — APPROVED** [review]
Thank you!

File: crates/sui-framework/sources/governance/genesis.move [hunk 1]
```diff
@@ -10,6 +10,7 @@ module sui::genesis {
     use sui::tx_context::TxContext;
     use sui::validator;
     use std::option;
+    use sui::vec_map;
```
Comment (emmazzz) [inline crates/sui-framework/sources/governance/genesis.move:13]: Do we need vec_map here?

**Comment by emmazzz** [comment]
You may need to re-generate the snapshots in `sui-config` and `sui-cost`.

File: crates/sui-framework/sources/governance/genesis.move (hunk 1 above)
Comment (laura-makdah) [reply crates/sui-framework/sources/governance/genesis.move]: removed!

**Review by akichidis — APPROVED** [review]
Changes make sense, thanks @laura-makdah !

**Pull request merged**